### ISSUE TITLE
 There is no create_diffusers_controlnet_model_from_ldm function in single_file_utils.py

### ISSUE THREAD

**Issue by meng-yuan321** [huggingface/diffusers#8645]
### Describe the bug

in diffusers/loaders/controlnet.py, there is 
from .single_file_utils import (
    create_diffusers_controlnet_model_from_ldm,
    fetch_ldm_config_and_checkpoint,
)

But, there is no create_diffusers_controlnet_model_from_ldm function in single_file_utils.py

### Reproduction

from .single_file_utils import (
    create_diffusers_controlnet_model_from_ldm,
    fetch_ldm_config_and_checkpoint,
)

### System Info

`diffusers`version:0.21.0
Platform: Linux-5.4.143-2-velinux1-amd64-x86 64-with-glibc2.31
Python version: 3.9.17
PyTorch version(GPu?):2.0.1+cu118(True)
Huggingface hub version: 0.23.4
Transformers version:4.25.0
Accelerate version: 0.31.0
xFormers version: 0.0.22

**Comment by sayakpaul** [comment]
Indeed. Cc: @DN6. It must have been caught in the tests too. Let's fix this.

**Comment by DN6** [comment]
This is a bug. Thanks for flagging. That file and import should be removed. I'll open a PR for it
